components/Ink/src/MDCInkView.m

```diff
@@ -133,9 +133,9 @@ - (void)setInkStyle:(MDCInkStyle)inkStyle {
   }
 }
 
-- (void)setInkColor:(UIColor *)inkColor {
+- (void)setInkColor:(nullable UIColor *)inkColor {
```

Comment:
Any reason to specify the nullability in the `.m` file? That's pretty rare in the project so I'm curious why it was added here.